### ISSUE TITLE
Fix keyboard accessibility of uberbar menu

### ISSUE THREAD

**Issue by dubrod** [modxcms/a11y#20]
For compliance with Web Content Accessibility Guidelines (WCAG)

The main drop down menus are not accessible by keyboard. This is a big problem -- it's a deal-breaker, really -- that affects blind users and sighted keyboard users, neither of which can use a mouse effectively. The menu is 100% inaccessible to these groups. The ideal would be to use an ARIA-based menu that allows users to tab into the group (the whole nav/menu group) and then tab past it without tabbing into the menu items at all, and use the arrow keys to navigate within the menu system itself. See the keyboard patterns outlined here:

http://www.w3.org/TR/wai-aria-practices/#menu
http://www.w3.org/TR/wai-aria-practices/#kbd_focus

See accessibility to-do list at https://dequeuniversity.com/modx-accessibility.html#uberbar
http://wet-boew.github.io/v4.0-ci/demos/menu/menu-en.html

**Comment by dubrod** [comment]
@rtripault this really is the most important item we need to tackle. proper aria roles and keyboard arrow navigation. not just the hacked together tab ability i have working b/c of this. https://github.com/modxcms/a11y/blob/master/manager/templates/a11y-theme/js/init-a11y.js#L44

**Comment by dubrod** [comment]
@theboxer 
the commit I just added has much better keyboard control to the ubermenu, but has some flaws.

1. I only attached it to the menu `modx-topnav` for a smaller test case
2. I had to break apart the function depending on the e.target so you will see on line 8 i hard coded the `<li>` names. I could not get an array string checking statement to work. 
3. I still need to handle some `null` returns better like if you reach the end and hit down or if you go back item 1 and hit up
4. It does not reach the 2nd level nav under Manage

**Comment by dubrod** [comment]
i believe this to finally be done !
